### DISCUSSION TITLE
Things to tackle after beta launch

### DISCUSSION THREAD

**Discussion by jasonappah** [acmutd/comet_planning#227] [General]
Use this as a tracking issue for issues we want to address and/or discuss after we launch.

**Comment by jasonappah** [comment]
To start:
- Creating Tailwind CSS tokens for sizes/colors (#226)
- Migrating away from MUI
- Integrating degree validation logic (#193, #169)
- Upgrade to React 18

**Comment by akevinge** [comment]
To add on to this:
- Update Firebase to v9, which includes tree shaking support
- Update Next to v12.3 for more SWC features, specifically import modularization and minifier support

The above would help address:
- #168  
- Optimization of all pages

**Comment by ZzRanger** [comment]
NextAuth
tRPC
React 18 Next 13
React query

**Comment by ZzRanger** [comment]
Linting+TS+Other stuff

1. Allow empty interfaces (eslint@typescript-eslint/no-empty-interface)
